docker/README.md

```diff
@@ -52,7 +52,7 @@ docker-compose restart        # Restart RackHD and ELK.
 It is recommended that you uncomment the `core` RackHD service before developing RackHD. This service doesn't need to run but it should be rebuilt before other RackHD images are built.
 
 ```
-$ ../scripts/reset_submodules.bash  # Fetch latest submodules.
+$ ../scripts/reset_submodules.bash  # Fetch latest submodules, otherwise you can set every submodule to the specific commit you want.
```

Comment:
”you can“, if not fetch latest, you can set every submodule to the specific commit you want.